gsy_framework/utils.py

```diff
@@ -99,7 +100,7 @@ def generate_market_slot_list(start_timestamp=None):
     @return: List with market slot datetimes
     """
     time_span = duration(days=PROFILE_EXPANSION_DAYS) \
-        if GlobalConfig.IS_CANARY_NETWORK \
+        if GlobalConfig.CONFIG_TYPE \
```

Comment:
I think that collaborations should not be included here, because they also have a valid `sim_duration`.

```suggestion
        if is_canary_network() \
```

Reply:
This is a bug, will fix it. Thanks